cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -57,6 +57,7 @@
     private final String mongoUsername;
```

Comment:
CHANGES_NEXT_RELEASE entry corresponding to the changes in this PR should be added (with a mention of the new configuration parameter being added)

Reply:
Fixed in cba084e